File: spring/prog_terminal_marittimo/src/main/java/progetto_terminal_marittimo/modal/autista.java
package progetto_terminal_marittimo.modal;

public class autista {
    private int ID;
    private String nome;
    private String cognome;
    private String email;

    public autista(int id, String nome, String cognome, String email) {
        this.ID = id;
        this.nome = nome;
        this.cognome = cognome;
        this.email = email;
    }

    public int getID() {
        return this.ID;
    }

    public String getNome() {
        return this.nome;
    }

    public String getCognome() {
        return this.cognome;
    }

    public String getEmail() {
        return this.email;
    }
}

File: pagine/js/autista.js
class Autista {
  constructor(id, nome, cognome, email) {
    this.id = id;
    this.nome = nome;
    this.cognome = cognome;
    this.email = email;
  }

  getID() {
    return this.id;
  }

  getNome() {
    return this.nome;
  }

  getCognome() {
    return this.cognome;
  }

  getEmail() {
    return this.email;
  }

  toString(){
    return "nome: "+this.nome+", cognome: "+this.cognome+", email: "+this.email;
  }
}
